ansible/group_vars/microk8s/btrix_values.j2

```diff
@@ -0,0 +1,15 @@
+ingress_class: "public"
+
+mongo_auth:
+  username: root
+  password: example
+
+ingress:
+  host: "microk8s.browsertrix.cloud"
```

Comment:
Should the host, email and signer.host be templates that are passed in from Ansible variables?